Rarime/Code/Managers/UserManager.swift

```diff
@@ -38,7 +38,7 @@ class UserManager: ObservableObject {
             
             self.user = try User.load()
             self.balance = 0
-            self.isPassportTokensReserved = AppUserDefaults.shared.isPassportTokensReserved
+            self.isPassportTokensReserved = false
```

Comment:
let's remove it from here and move it directly to home view

also, the reserve card should not be shown if balance is not loaded